admin/test/test_runner.py

```diff
@@ -105,6 +105,20 @@ def test_unregisters_killer_failure(self):
 
     def test_process_success(self):
         """
+        If the process ends with a success, the returned deferred fires with
+        the reason.
+        """
+
+        reactor = ProcessCoreReactor()
+        d = run(reactor, ['command', 'and', 'args'])
+        [process] = reactor.processes
+
+        expected_failure = Failure(ProcessDone(0))
+        process.processProtocol.processEnded(expected_failure)
+        self.successResultOf(d),
```

Comment:
Actually, the reason for success? That means the docstring should be fixed and this comma should be removed.